src/natcap/invest/globio.py

```diff
@@ -975,7 +982,7 @@ def _collapse_infrastructure_op(*infrastructure_array_list):
         (x, 1) for x in aligned_infrastructure_target_list]
     pygeoprocessing.raster_calculator(
         infra_filename_band_list, _collapse_infrastructure_op,
-        infrastructure_path, gdal.GDT_Byte, infrastructure_nodata)
+        infrastructure_path, gdal.GDT_Int32, infrastructure_nodata)
```

Comment:
I commented on this above too, just checking why it's necessary to change this to an int32 when the result is a 0/1 mask?

Reply:
It's not, should stay a Byte since the out nodata is -1 and results will be 0 and 1.

Reply:
Ahh got it, I missed that there is a nodata possibility there. I feel like making the initial raster a `numpy.uint8` type, then setting the nodata value to "2" or "255" is the solution that keeps everything in 8 bit space?

Reply:
Nodata value is now 255 here: https://github.com/natcap/invest/pull/186/commits/86dab80e89b22c452e5da5b2efec78decbfa62cb